src/frontend/src/components/items/MenuLinks.tsx

```diff
@@ -45,13 +45,13 @@ function ConditionalDocTooltip({
 
 export function MenuLinks({
   links,
-  highlighted
+  highlighted = false
 }: {
   links: MenuLinkItem[];
   highlighted?: boolean;
 }) {
   const { classes } = InvenTreeStyle();
-  highlighted = highlighted || false;
+  highlighted = highlighted ?? false;
```

Comment:
Mutating props is a really bad practice, props are immutable in react. This should be a second variable, and if used inside a dependency array it needs to be wrapped in a useMemo. Basically this could also be removed, as the default value is already assigned at the function definition.

Reply:
@wolflu05 I have simplified this now, thanks for the insight